src/Nancy.Tests/Unit/Diagnostics/CustomInteractiveDiagnosticsFixture.cs

```diff
@@ -127,7 +127,7 @@ public void Should_return_main_page_with_valid_auth_cookie()
             // When
             var result = browser.Get(diagsConfig.Path + "/interactive/providers/", with =>
                 {
-                    with.Cookie(DiagsCookieName, this.GetSessionCookieValue("password"));
+                    with.Cookie(DiagsCookieName, Nancy.Helpers.HttpUtility.UrlEncode(this.GetSessionCookieValue("password")));

```

Comment:
I'd rather add a `using Nancy.Helpers` (or `using Helpers` might even do it) and get rid of the full name for these.